modules/Bio/EnsEMBL/Compara/RunnableDB/OrthologQM/PrepareOrthologs.pm

```diff
@@ -236,7 +171,7 @@ sub _reusable_homologies {
     }
 
     # now, we split the homologies into reusable and non-reusable (new)
```

Comment:
I'm not 100% sure, but I have a feeling after removing `@reusables` this comment is not longer accurate: you are now just detecting the non-reusable (new) homologies, right?